qtcodes/fitters/rotated_surface.py

```diff
@@ -191,14 +212,15 @@ def _specify_virtual(self) -> Dict[str, List[TQubit]]:
         virtual: Dict[str, List[TQubit]] = {}
         virtual["X"] = []
         virtual["Z"] = []
-        for j in range(0, self.params["d"], 2):
+        for j in range(0, self.params["d"][1], 2):
```

Comment:
I'd say we should probably be consistent and capture these to local variables `dw` and `dh` as you have done above.